src/app/ssmt/header/header.service.ts

```diff
@@ -98,8 +113,18 @@ export class HeaderService {
       useBaselineProcessSteamUsage: [obj.useBaselineProcessSteamUsage]
     });
 
+    // only add validation if pressureMin/Max constraint fields have value
+    if (pressureMin) {
```

Comment:
these should probably be if `pressureMin != undefined` right now it won't pass if `pressureMin == 0`